pkg/agent/proxy/proxier.go

```diff
@@ -476,19 +491,19 @@ func (p *proxier) installServices() {
 					// of the Service are different, install two groups. One group has all Endpoints, the other has only
 					// local Endpoints.
 					groupID := p.groupCounter.AllocateIfNotExist(svcPortName, true)
-					if err = p.ofClient.InstallServiceGroup(groupID, affinityTimeout != 0, localEndpointUpdateList); err != nil {
```

Comment:
Will NodePort/LoadBalancer type be used for a multi-cluster Service?

I ask it because I think when a local Service is working as an Endpoint in the multi-cluster Service, the Endpoint should not be a local Endpoint.

Reply:
No, we didn't support NodePort/LoadBalancer type in MC. But both NodePort/LoadBalancer have ClusterIP assigned, we may still treat them as ClusterIP according to KEP1645.